ipam/http.go

```diff
@@ -199,6 +199,6 @@ func (alloc *Allocator) HandleHTTP(router *mux.Router, defaultSubnet address.CID
 	})
 
 	router.Methods("GET").Path("/ipinfo/tracker").HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-		fmt.Fprintf(w, tracker)
+		fmt.Fprintf(w, alloc.tracker.String())
```

Comment:
What is this used for?  I can't find anything hitting this endpoint in the repo.

Reply:
By [detect_awsvpc](https://github.com/weaveworks/weave/blob/master/weave#L900).